models/servicenow__user_aggregated.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(enabled=var('servicenow__using_roles', False)) }}
+
+with sys_user as (
+    
+    select *
+    from {{ ref('stg_servicenow__sys_user') }}
+),
+
+user_grmember as (
+
+    select * 
+    from {{ ref('stg_servicenow__sys_user_grmember') }}
+),
+
+user_has_role as (
+
+    select * 
+    from {{ ref('stg_servicenow__sys_user_has_role') }}
+),
+
+user_role as (
+
+    select * 
+    from {{ ref('stg_servicenow__sys_user_role') }}
+),
+
+user_group as (
+
+    select * 
+    from {{ ref('stg_servicenow__sys_user_group')}}
+),
+
+user_aggregates as (
+
+    select
+        sys_user.sys_user_id,
+        sys_user.source_relation,
+        {{ dbt.listagg(measure="user_grmember.sys_user_group_id") }} as sys_user_group_ids,
```

Comment:
Just calling out that I'm using listagg to aggregate these strings, but cautious as we ran into limit issues in fivetran log with this

Reply:
Thanks for the input!